PgpWordList/PgpWordAggregationBuilder.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Messerli.PgpWordList
+{
+    public class PgpWordAggregationBuilder
+    {
+        public const string DefaultSeparator = "-";
+
+        private readonly List<byte> _byteList = new List<byte>();
+
+        private string _separator = DefaultSeparator;
+
+        public PgpWordAggregationBuilder SetSeparator(string separator)
+        {
+            _separator = separator;
+            return this;
+        }
+
+        public PgpWordAggregationBuilder Add(byte @byte)
+        {
+            _byteList.Add(@byte);
+            return this;
+        }
+
+        public PgpWordAggregationBuilder Add(byte[] byteArray)
+        {
+            _byteList.AddRange(byteArray);
+            return this;
+        }
+
+        public PgpWordAggregation Build()
+            => new PgpWordAggregation(
+                string.Join(
+                    _separator,
+                    _byteList.Aggregate(new List<string>(), ByteToWord)));
+
+        private static List<string> ByteToWord(List<string> list, byte @byte)
+        {
+            list.Add(
+                list.Count % 2 == 0
```

Comment:
Extract this into an `IsEven` function

Reply:
Addressed in https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/048fb2d105c4bf7d629d2da2b6273312daae5708